### DISCUSSION TITLE
SVG image in Button ignoring Opacity

### DISCUSSION THREAD

**Discussion by gwynroberts** [wieslawsoltes/Svg.Skia#151] [Q&A]
If I add a TextBlock and an SVG Image to a Button, and set the button's opacity, the opacity is applied to the TextBlock but not to the image. I'd expect the opacity to apply to the image too.

@wieslawsoltes could this be fixed?

Note, I can work around the issue by setting an opacity mask for the image using the button's background color. However, would be nice if it was fixed.

**Comment by danipen** [comment]
@wieslawsoltes any ideas about this?

**Reply by danipen** [reply]
I think we're experiencing this right? https://github.com/wieslawsoltes/Svg.Skia/discussions/82

**Reply by danipen** [reply]
@wieslawsoltes what needs to be fixed in Avalonia side in order to get it working here?

**Reply by wieslawsoltes** [reply]
https://github.com/AvaloniaUI/Avalonia/blob/ec6cd48de9408cb39b12f52cf4af708a93e42988/src/Skia/Avalonia.Skia/SkiaOptions.cs#L26
